ckan/logic/action/update.py

```diff
@@ -777,10 +783,151 @@ def _group_or_org_update(context, data_dict, is_org=False):
 
     if not context.get('defer_commit'):
         model.repo.commit()
+        # (canada fork only): background search index rebuilding
+        #TODO: upstream contrib!!
+        if (plugins.toolkit.asbool(plugins.toolkit.config.get('ckan.search.reindex_after_group_update', False))
```

Comment:
```suggestion
        if (plugins.toolkit.asbool(plugins.toolkit.config.get('ckan.search.reindex_after_group_or_org_update', False))
```

Reply:
Need to add this to the config_declaration.yaml too

Reply:
Don't think we have a config_declaration in this fork yet as is a 2.10 thing? But yeah will need that upstream and in our 2.10 branch